force-app/main/default/classes/SOQL.cls

```diff
@@ -85,9 +96,10 @@ public with sharing class SOQL {
      * @return A map of Task Ids to Task records.
      * Update the corresponding test class and testGetTasksDueNextWeek method to cover this method
      */
-    //[public | private ] [static] data_type method_name (input parameters) {
-    //  The body of the method
-    //}
+    public static Map<Id,Task> getTasksDueNextWeek () {
+        Map<Id,Task> taskMap = new Map<Id,Task>([Select Id, ActivityDate From Task Where ActivityDate = NEXT_N_DAYS:7]);
```

Comment:
Good job with the date formulas, We didnt go over that but you found the proper format.